packages/maker/src/Maker/FrontendApiTypeMaker.php

```diff
@@ -0,0 +1,209 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\MakerBundle\Maker;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\Mapping\ClassMetadata;
+use Override;
+use Shopsys\MakerBundle\Utils\EntityChoiceHelper;
+use Symfony\Bundle\MakerBundle\ConsoleStyle;
+use Symfony\Bundle\MakerBundle\DependencyBuilder;
+use Symfony\Bundle\MakerBundle\Generator;
+use Symfony\Bundle\MakerBundle\InputConfiguration;
+use Symfony\Bundle\MakerBundle\Maker\AbstractMaker;
+use Symfony\Bundle\MakerBundle\Str;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Yaml\Yaml;
+
+class FrontendApiTypeMaker extends AbstractMaker
```

Comment:
I'm not sure why, but when I generated the entity Asd, then api type. then I removed all files and generate the entity again with different fields. 

When I then ran the frontend api type generation it failed on the error 
```
In RuntimeReflectionProperty.php line 29:

  Property App\Model\Asd\Asd::$catnum does not exist
  ```

catnum was the field generated the first time, second time i did not generate it. 

I had to clean cache

Reply:
yes, I have encountered that as well, I really do not why. I am not sure whether it is worth adding the cache clear call at the beginning of the command :thinking: IMHO it is not